docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# http://nexb.com and https://github.com/nexB/scancode.io
```

Comment:
I reckon you copied this from ScanCode.io... but there is no need to carry the scancode.io references